src/main/java/seedu/address/model/person/Person.java

```diff
@@ -35,18 +36,21 @@ public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tag
         this.email = email;
         this.address = address;
         this.remark = Remark.EMPTY_REMARK;
+        this.birthday = Birthday.EMPTY_BIRTHDAY;
         this.tags.addAll(tags);
     }
     /**
      * Every field must be present and not null. with non-empty remark
      */
-    public Person(Name name, Phone phone, Email email, Address address, Remark remark, Set<Tag> tags) {
+    public Person(Name name, Phone phone, Email email, Address address, Remark remark, Birthday birthday,
```

Comment:
I would suggest to keep the orginal constructor, and initialise the birthday field to EMPTY, and add an overloaded constructor to handle the case when birthday is specified at creation, since there would be no default minimal constructor for the case when birthday is not specified upon adding.

Reply:
Alright, fixed this